### DISCUSSION TITLE
Add my onChange fn in MRT_GlobalFilterTextField component

### DISCUSSION THREAD

**Discussion by dangkhoa99** [KevinVandy/material-react-table#885] [Ideas]
Hello, @KevinVandy.

I want to add my onChange function into the existing onChange of the MRT_GlobalFilterTextField component. But now, my custom fn override the existing onChange. My onChange function wants to reset rowSelection in the table after filtering.

Can you add the my ideas in MRT_GlobalFilterTextField component ?

![image](https://github.com/KevinVandy/material-react-table/assets/56160839/c5b150ad-afb8-4ab1-818d-8e0b2d944e40)

**Comment by MistyKuu** [comment]
I guess you can use onChangeCapture which is the same event but it doesn't override anything
